plugins/generator-1.18.2/forge-1.18.2/mappings/defaultfeatures.yaml

```
@@ -32,5 +33,5 @@ TallSeagrassLush: DefaultSeagrass
 WarmFlowers: WarmFlowers
 
 #Legcay

```

```suggestion
#Legacy
```
NOTE: I didn't check the other mappings.